.env.dist

```diff
@@ -9,23 +10,23 @@ LOGGER_LEVEL=debug
 LOGGER_TYPE=json
 
 ###DB config
-DB_HOST=localhost
-DB_PORT=5432
-DB_USER=postgres
-DB_PASSWORD=foobar
-DB_NAME=test
-DB_SSL_MODE=disable
-DB_MAX_IDLE_CONNS=5
-DB_MAX_OPEN_CONNS=10
-DB_CONN_MAX_LIFETIME=3600
 
+GORM_DSN="host=localhost port=5432 user=postgres dbname=test password=foobar prefer_simple_protocol=1 sslmode=disable"
```

Comment:
Changed gorm connection to the DSN format. It will be easier to work with the master-slave replication if we have a format as DSN for the each connection